examples/jsm/utils/SceneUtils.js

```diff
@@ -143,13 +143,17 @@ function reduceVertices( object, func, initialValue ) {
 
 				for ( let i = 0, l = position.count; i < l; i ++ ) {
 
-					vertex.fromBufferAttribute( position, i );
+          if ( child.isMesh ) {
 
-					if ( child.isSkinnedMesh ) {
+            child.getVertexPosition( i, vertex );
 
-						child.boneTransform( i, vertex );
+          } else {
 
-					} else {
+            vertex.fromBufferAttribute( position, i );
+
+          }
+
+					if ( !child.isSkinnedMesh ) {
```

Comment:
Linting:

The indentation is incorrect.

Also, it should be `! child.isSkinnedMesh` -- with a space.

Reply:
Oh, good call; I thought the auto-linting would catch that kind of thing now.